src/wallet/rpcwallet.cpp

```diff
@@ -732,7 +732,7 @@ static UniValue getbalance(const JSONRPCRequest& request)
                 "thus affected by options which limit spendability such as -spendzeroconfchange.\n",
                 {
                     {"dummy", RPCArg::Type::STR, RPCArg::Optional::OMITTED_NAMED_ARG, "Remains for backward compatibility. Must be excluded or set to \"*\"."},
-                    {"minconf", RPCArg::Type::NUM, /* default */ "0", "Only include transactions confirmed at least this many times."},
+                    {"minconf", RPCArg::Type::NUM, /* default */ "0", "Must be zero"},
```

Comment:
It's very confusing for this to be `0`, because it won't actually show unconfirmed incoming transactions, only unconfirmed change. It's not even really `0` by default, but `1`, see a few lines below. Since we can't drop it and can't rename it to `dummy2`, let's add a comment: `This is ignored and only remains for backward compatibility. Must be excluded or set to zero.`